ksql-common/src/test/java/io/confluent/ksql/util/SchemaUtilTest.java

```diff
@@ -16,323 +16,31 @@
 
 package io.confluent.ksql.util;
 
-
-import org.apache.kafka.connect.data.Field;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.sameInstance;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-import java.util.HashMap;
-import java.util.Optional;
-
 public class SchemaUtilTest {
 
-  Schema schema;
-
-  @Before
```

Comment:
Why have we removed all the tests in here and put them in another class of the same name in a different package? There should only be 1 `SchemaUtilTest` class

Reply:
All tests are in one class now.